agr_literature_service/api/models/reference_mod_md5sum_model.py

```diff
@@ -0,0 +1,75 @@
+"""
+reference_mod_md5sum_model.py
+===============
+"""
+
+
+from datetime import datetime
+import pytz
+from sqlalchemy import (Column, ForeignKey, Integer, String, DateTime)
+from sqlalchemy.orm import relationship
+from agr_literature_service.api.database.base import Base
+from sqlalchemy.schema import Index
+
+
+class ReferenceModMd5sumModel(Base):
+    __tablename__ = "reference_mod_md5sum"
+
+    reference_mod_md5sum_id = Column(
+        Integer,
+        primary_key=True,
+        autoincrement=True
+    )
+
+    reference_id = Column(
+        Integer,
+        ForeignKey("reference.reference_id", ondelete="CASCADE"),
+        nullable=False,
+        index=True
+    )
+
+    reference = relationship(
+        "ReferenceModel",
+        back_populates="reference_mod_md5sum"
+    )
+
+    mod_id = Column(
+        Integer,
+        ForeignKey("mod.mod_id", ondelete="CASCADE"),
+        index=True,
+        nullable=True,
+    )
+
+    mod = relationship(
+        "ModModel",
+        back_populates="reference_mod_md5sum"

```

Comment:
we may need to remove this back_populates as it might generate an infinite chain of objects. I'd like to see more tests with model objects to try this